Week-2/Html Basics/hometownName.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hometown</title>
+</head>
+<body>
+
+    <h1>Udaipur, Rajasthan</h1>
```

Comment:
The hierarchy of header levels should be consistent and follow a logical order. For example, `<h2>` elements should be followed by `<h3>` elements, not by another `<h2>`. In this code, "Location" and "About" are both at the same level, which can be confusing for screen readers and SEO. You should use a proper hierarchy like `<h2> `for main sections and `<h3>` for subsections.